cli/install/install.go

```diff
@@ -979,6 +955,14 @@ func installTarantoolEE(binDir string, includeDir string, installCtx InstallCtx,
 	distfiles string, cliOpts *config.CliOpts) error {
 	var err error
 
+	// Check bin and header directories.
+	if binDir == "" {
+		return fmt.Errorf("binDir is not set, check %s", configure.ConfigName)
```

Comment:
A user knows nothing about what `binDir` is. Why do we print variable name here? The same is for `includeDir`.

Reply:
changed to .yaml names